receiver/splunkhecreceiver/config.go

```diff
@@ -21,6 +22,9 @@ type Config struct {
 	confighttp.ServerConfig `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
 	splunk.AccessTokenPassthroughConfig `mapstructure:",squash"`
+
+	Ack `mapstructure:",squash"`
```

Comment:
Should we be adding this configuration option to the README? 

I see the `ack` extension isn't currently being deployed by contrib, should it be included in releases of contrib before including this functionality?

Reply:
I just updated the readme to include ack. Would I need to modify something to make the ack exntesion be deployed by cotrib?

Reply:
PR for ack extension release https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32308
Having some troubles getting the checks to pass though :(